library/alloc/tests/slice.rs

```diff
@@ -1722,7 +1722,7 @@ fn panic_safe() {
     let mut rng = thread_rng();
 
     // Miri is too slow
-    let lens = if cfg!(miri) { (1..10).chain(20..21) } else { (1..20).chain(70..MAX_LEN) };
+    let lens = if cfg!(miri) { (1..10).chain(0..0) } else { (1..20).chain(70..MAX_LEN) };
```

Comment:
I added the "20" case without measuring, which [clearly was a mistake](https://github.com/RalfJung/miri-test-libstd/issues/2). So let's remove it again.

(We have to `chain` to make this type-check.)

Reply:
> (We have to `chain` to make this type-check.)

Perhaps this rationale is best left in a comment?

Reply:
Good point, done.